docs/guideis/workflow.md

```diff
@@ -0,0 +1,29 @@
+
+
+
+1. Choose or create an [Issue](https://github.com/open-marketplace-applications/dshop/issues).
+
+2. Check if there is a PR - if now, if not create one:
+
+    2.1 Go to your IDE (We recommend Visial Studio Code)
+    2.2 Create a new branch, use a tag (`feature`, `fix`, or `docs`) and a descriptive name. Example: `feature/<descriptive-name-of-your-feature>`
+    2.3 Push your new branch and create a PR.
+    
+    2.4 Add a description, assign yourself and add some labels.
+
+        Tip: Add this to the issue description, to close the issue automatically after merge:
+
+        Example:
+        ```bash
+        closes Issue #<issue_number>
+        ```
+
+        Example:
+        ```bash
+        closes Issue #42
+        ```
+    2.5 Create it as a `Draft Pull Request`. (Press the small arrow on the "Create pull request" Button).
```

Comment:
Numbered list in this small example case is somehow too much, in my opinion. Check my final request of changes comment, that will possibly help, to structure the docs more fancy and faster - maybe also put those hints and the plugin link from my comment into the document itself, as side-note at the bottom ? Could imagine that other devs, that are not that much familiar with markdown, will benefit from having that info and plugin link inside this workflow .md file. :rocket: